backend/api/v1/rollout_service_task.go

```diff
@@ -152,15 +152,15 @@ func getTaskCreatesFromCreateDatabaseConfig(ctx context.Context, s *store.Store,
 		}
 
 		payload := api.TaskDatabaseCreatePayload{
-			SpecID:       spec.Id,
-			ProjectID:    project.UID,
-			CharacterSet: c.CharacterSet,
-			TableName:    c.Table,
-			Collation:    c.Collation,
-			Environment:  c.Environment,
-			Labels:       labelsJSON,
-			DatabaseName: databaseName,
-			SheetID:      sheet.UID,
+			SpecID:        spec.Id,
+			ProjectID:     project.UID,
+			CharacterSet:  c.CharacterSet,
+			TableName:     c.Table,
+			Collation:     c.Collation,
+			EnvironmentID: strings.TrimPrefix(c.Environment, "environments/"),
```

Comment:
Use `common.EnvironmentNamePrefix` instead?
```suggestion
			EnvironmentID: strings.TrimPrefix(c.Environment, common.EnvironmentNamePrefix),
```

Reply:
Thanks!